server/index.js

```diff
@@ -37,32 +21,29 @@ var urlencodedParser = bp.urlencoded({ extended: false });
 app.use(urlencodedParser);
 
 //initialize session
-app.use(session({secret: 'secret'}));
-
-//serve static assets
-app.use(express.static('../Public'));
-
-
-var restrict = function(req, res, next) {
-  console.log('Inside restrict: ', req.session);
-  if (req.session.username && req.session.password) {
-    next();  
-  } else {
-    res.redirect('/login');
-  }
-};
-
-
-//-------------------------- ROOT -------------------------//
-app.get('/', restrict, function(req, res) {
-  res.sendfile(path.resolve('../Public/index.html'));
```

Comment:
Now we can serve static assets with:

``
res.sendfile('Public/index.html');
``